test/cpp/qps/json_run_localhost.cc

```diff
@@ -0,0 +1,89 @@
+/*
```

Comment:
nit: only json_run_localhost seems to be a confusing name for the binary. Pls consider prefixing with qps_ or something in that sense.